src/client/containers/PullRequestDiff/__tests__/index.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai'
+import {
+  getData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getData', () => {
```

Comment:
Let's name it more specifically, e.g. `PullRequestDiff  selectors`, then `getData should return ...`, it's becoming already too difficult to locate test source ...(we should go through test suites and name properly all of them)